helpers/helpers.js

```diff
@@ -48,3 +48,61 @@ export function toKebabCase(string) {
     .join('')
     .toLowerCase();
 }
+
+export function formatCurrency(amountInCents) {
+  const formatter = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 2,
+  });
+
+  // Convert cents to dollars by dividing by 100
+  return formatter.format(amountInCents / 100);
+}
+
+export function formatPhoneNumberToE164(phoneNumber, countryCode = '1') {
+  // Remove all non-numeric characters
+  const digits = phoneNumber.replace(/\D/g, '');
+
+  // Ensure the phone number has at least 10 digits (US standard)
+  if (digits.length < 10) {
+    throw new Error('Invalid phone number. Must contain at least 10 digits.');
+  }
+
+  // Add the country code prefix if not already included
+  if (digits.startsWith(countryCode)) {
+    return `+${digits}`; // Already in E.164 format
+  }
```

Comment:
i think this poses a problem for international numbers, but i think its more of an edge case to kick down the road